### ISSUE TITLE
Cant seem to get selectData to work

### ISSUE THREAD

**Issue by DavidCarl** [polozin/date-and-time-picker#1]
Hello

I tried to use selectData to get custom start time to work, but it for some reason it does nothing for me.

I included a snippet of how I set it up.

`    $('#picker-start').dateTimePicker({
        positionShift: { top: 0, left: 0 },
        title: "Select Date and Time",
        buttonTitle: "Select",
        selectData: "2018-09-25 08:06",
    });`

Anything you seem to be misplaced?

Thanks in advance :)

**Comment by justrustyspoons** [comment]
Yeah I had this issue as well. Looks like this plugin is not maintained and isn't fully working. I'd find a different plugin if I were you!

**Comment by DavidCarl** [comment]
I actually got it working and forgot all of this issue.... Anything you need help to do aswell?

**Comment by justrustyspoons** [comment]
Nah, don't worry, the plugin is not maintained and is lacking some key features for me. I'll just use a vanilla solution or write my own. Thanks!

**Comment by Haegge** [comment]
> I actually got it working and forgot all of this issue.... Anything you need help to do aswell?
@DavidCarl

Could you remember how you got it to work? The Plugin fits perfect to my needs, only the selectData Problem is drivin me nuts.

Thanks in Advance :)

**Comment by DavidCarl** [comment]
> Could you remember how you got it to work? The Plugin fits perfect to my needs, only the selectData Problem is drivin me nuts.
> 
> Thanks in Advance :)

I cannot remember on top of my head, but im pretty sure I got the source code somewhere. I will take a look around :)

**Comment by DavidCarl** [comment]
If its about setting a start time past the current time im pretty sure [this](https://github.com/prolike/timereg/blob/8ad21417f2a87ef7696fb25bc01f3c65f14b2e93/python_lib/flask/static/js/datetimepicker.js#L20) was the solution.

Out comment the lines I did.

It was my CS internship project I did at the company I work at. If that isn't what you meant to do I am not sure what you want to achieve.

It also seems that #4 might be a better solution for my fix.

**Comment by Haegge** [comment]
Thank you very much, that did it for me. It was obvious but couldn't see the forest for the trees.
